root/statistics/Index.js

```diff
@@ -0,0 +1,1039 @@
+          <td />
+        </tr>
+        <tr>
+          <th />
+          <th colSpan="2">{l_statistics('Mediums with no disc IDs:')}</th>
+          <td>{formatCount(stats.data['count.medium.0discids'], $c)}</td>
+          <td>{formatPercentage(stats.data['count.medium.0discids'] / stats.data['count.medium'], 1, $c)}</td>
+        </tr>
+        <tr>
+          <th />
+          <th colSpan="2">{l_statistics('Mediums with at least one disc ID:')}</th>
+          <td>{formatCount(stats.data['count.medium.has_discid'], $c)}</td>
+          <td>{formatPercentage(stats.data['count.medium.has_discid'] / stats.data['count.medium'], 1, $c)}</td>
+        </tr>
+        {oneToNine.map(num => (
+          <tr key={num}>
+            <th />
+            <th />
+            <th>{ln_statistics('with {num} disc ID:', 'with {num} disc IDs:', num, {__react: true, num: num})}</th>
+            <td>{formatCount(stats.data['count.medium.' + num + 'discids'], $c)}</td>
+            <td>{formatPercentage(stats.data['count.medium.' + num + 'discids'] / stats.data['count.medium.has_discid'], 1, $c)}</td>
+          </tr>
+        ))}
+        <tr>
+          <th />
+          <th />
+          <th>{l_statistics('with 10 or more disc IDs:')}</th>
+          <td>{formatCount(stats.data['count.medium.10discids'], $c)}</td>
+          <td>{formatPercentage(stats.data['count.medium.10discids'] / stats.data['count.medium.has_discid'], 1, $c)}</td>
+        </tr>
+      </tbody>
+    </table>
+
+    <h2>{l_statistics('Release Groups')}</h2>
+    <table className="database-statistics">
+      <tbody>
+        <tr className="thead">
+          <th colSpan="4">{l_statistics('Primary Types')}</th>
+        </tr>
+        <tr>
+          <th colSpan="2">{l_statistics('Release Groups:')}</th>
+          <td>{formatCount(stats.data['count.releasegroup'], $c)}</td>
+          <td />
+        </tr>
+        {((Object.values(primaryTypes): any): $ReadOnlyArray<ReleaseGroupTypeT>).map(primaryType => (
+          <tr key={primaryType.gid}>
+            <th />
+            <th>{l_statistics(primaryType.name, {__react: true})}</th>
+            <td>{formatCount(stats.data['count.releasegroup.primary_type.' + primaryType.id], $c)}</td>
+            <td>{formatPercentage(stats.data['count.releasegroup.primary_type.' + primaryType.id] / stats.data['count.releasegroup'], 1, $c)}</td>
+          </tr>
+        ))}
+        <tr className="thead">
+          <th colSpan="4">{l_statistics('Secondary Types')}</th>
+        </tr>
+        <tr>
+          <th colSpan="2">{l_statistics('Release Groups:')}</th>
+          <td>{formatCount(stats.data['count.releasegroup'], $c)}</td>
+          <td />
+        </tr>
+        {((Object.values(secondaryTypes): any): $ReadOnlyArray<ReleaseGroupSecondaryTypeT>).map(secondaryType => (
+          <tr key={secondaryType.gid}>
+            <th />
+            <th>{l_statistics(secondaryType.name, {__react: true})}</th>
```

Comment:
For some reason, secondary types are not localized anymore here. The same goes for label types, work types, work attributes, area types, place types, series types, instrument types, and event types.

Reply:
I changed this (and the Formats page) to use lp_attributes, and the relationships page to use l_relationships. AFAICT those were the only untranslated ones.

Reply:
Yes about the Overview tab, but the same issue appears for some linking phrases in the Relationships tab.

Reply:
I already changed that too - if it only happens to some, it might just be they're untranslated (the file for German at least had a ton of untranslated ones)

Reply:
I compared with (previous) beta, with French.